ir/function.cpp

```diff
@@ -48,6 +47,40 @@ JumpInstr::it_helper BasicBlock::targets() const {
   return {};
 }
 
+void BasicBlock::replaceTargetWith(const BasicBlock *from,
+                                   const BasicBlock *to) {
+  Instr &j = this->back();
+  if (auto br = dynamic_cast<Branch*>(&j)) {
```

Comment:
can you move this code to jump instruction pls?
create a virtual method that then gets specialized in branch, switch.
This is so we don't forget about this function once we add more CFG instructions.

Reply:
nice suggestion.